01-JS-DOM-APIs/css/style.css

```diff
@@ -0,0 +1,47 @@
+/*--- BOX PROPERTIES ---*/
+.hidden {
+  opacity: 0;
+}
+
+.results {
+  position: absolute;

```

Comment:
css rules are grouped by categories (box properties, box borders, ...) and then alphaordered

> **Order properties: box, border, background, text & other**
Always order your properties grouping by categories. Box properties (Position, layout, display), box borders, background properties, text properties (font size, family, weight, etc), and so on.